app/build.gradle

```diff
@@ -66,8 +74,9 @@ dependencies {
     compile 'com.facebook.android:facebook-android-sdk:4.6.0'
     compile 'com.parse:parsefacebookutils-v4-android:1.10.3@aar'
 
-    // Google Places API
+    // Google
     compile 'com.google.android.gms:play-services-places:9.8.0'
+    compile 'com.google.android.gms:play-services:9.8.0'
```

Comment:
should we remove the places specific one on the line above if we have this import now?